src/auth/entities/user.entity.ts

```
@@ -130,4 +134,4 @@ export class User {
   })
   @Column({ type: 'varchar', length: 100, nullable: true, unique: true })
   walletAddress?: string;
-}
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)